rust/syntect-plugin/src/main.rs

```diff
@@ -222,10 +222,10 @@ impl<'a> Syntect<'a> {
     /// Checks if a newline has been inserted, and if so inserts whitespace
     /// as necessary.
     fn do_indentation(&mut self, view: &mut MyView, start: usize,
-                      end: usize, text: &str) -> Option<u64> {
+                      end: usize, text: &str, undo_group: Option<usize>) {
```

Comment:
related to my previous comment: we know in this case that we always want an undo group, so we don't need to pass this through.